### PR TITLE
Adds cryptodevhub.io

### PR DESCRIPTION
**Pull request by prabhpreet-code**
[Please describe your pull request here]

Checklist
------------

* [ ] Each link description starts with an uppercase character and ends with a period.<br>Example: `solc-js - JavaScript bindings for the compiler.`
* [ ] Drop all the `A` / `An` prefixes in the descriptions.
* [ ] Avoid using the word `Solidity` in the description.

### CODE REVIEW

**Comment by prabhpreet-code** [comment]
updated readme file

File: README.md [hunk 1]
```diff
@@ -81,6 +82,7 @@
 - [Capture the Ether](https://capturetheether.com/) - Game in which you hack Ethereum smart contracts to learn about security.
 - [Ethereum Smart Contract Security Best Practices](https://consensys.github.io/smart-contract-best-practices/) - General security philosophy, known attacks, and sample code.
 - [Not So Smart Contracts](https://github.com/crytic/not-so-smart-contracts) - Examples of common vulnerabilities, including code from real smart contracts.
+- [Best practices for smart contract development](https://yos.io/2019/11/10/smart-contract-development-best-practices/)- Best practices for smart contract development
```
Comment (bkrem) [inline README.md:85]: This entry already exists [here](https://github.com/bkrem/awesome-solidity/pull/69/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R65), originally added by the author himself.
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -73,6 +73,7 @@
 - [Solidity and Vyper cheat sheet](https://reference.auditless.com/cheatsheet) - Review syntax of both languages side-by-side.
 - [Solidity Developer Blog](https://soliditydeveloper.com/blog) - Concepts, guides, design patterns and more.
 - [Clean Contracts](https://www.wslyvh.com/clean-contracts/) - Developer guide to writing clean smart contract code.
+- [Getting started with blockchain to advance solidity](cryptodevhub.io)-from starting in blockchain to advance solidity - all the resources at one point
```
Comment (bkrem) [inline README.md:76]: Nice addition thank you!

Please make sure to follow the checklist in the PR template for new entries. The URL also needs to be absolute (`https://...`) to work as expected in markdown. 

I've made some changes to account for the above:
```suggestion
- [cryptodevhub.io](https://cryptodevhub.io/) - Community-driven effort to unite like-minded people interested in Blockchain- and Crypto Technologies.
```

**Review by bkrem — CHANGES_REQUESTED** [review]
Thank you for adding cryptodevhub.io 🙏 

I've added some suggested changes, can merge once these are resolved 👍

**Review by bkrem — APPROVED** [review]

**Pull request merged**